recipe/meta.yaml

```
@@ -36,10 +37,12 @@ test:
   imports:
     - robyn
   commands:
-    - pytest -vvv --capture=tee-sys  integration_tests
+    - ls -laR --color
+    - pwd
+    - pytest -vvv --capture=tee-sys
```

Not sure if this will help but worth a try:

```suggestion
    - pytest -vvv --capture=tee-sys --pyargs robyn
```